packages/loot-core/migrations/1720664867241_add_payee_favorite.sql

```diff
@@ -0,0 +1,5 @@
+BEGIN TRANSACTION;
+
+ALTER TABLE payees ADD COLUMN favorite INTEGER DEFAULT 0 DEFAULT FALSE;
```

Comment:
Are there two defaults on purpose?  Its working so its not that big of a deal I don't think.